packages/expo-file-system/ios/EXFileSystem/EXSessionTasks/EXSessionResumableDownloadTaskDelegate.m

```diff
@@ -47,9 +42,17 @@ - (void)URLSession:(NSURLSession *)session task:(NSURLSessionTask *)task didComp
                   [NSString stringWithFormat:@"Unable to download file: %@", error.description],
                   error);
     }
+    
+    [_manger unregisterTask:_uuid];
```

Comment:
Is there a situation when both `- (void)URLSession:task:didCompleteWithError:` and `- (void)URLSession:downloadTask:didFinishDownloadingToURL` are called? Is it possible to put `unregisterTask:` only in one of the callbacks (eg. if `didCompleteWithError:` is always called when the task completes we could put `unregisterTask:` there)?